build_for_ultimaker.sh

```diff
@@ -6,61 +6,51 @@
 # Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
 #
 
-ARCH="${ARCH:-armhf}"
-CI_REGISTRY_IMAGE="${CI_REGISTRY_IMAGE:-registry.gitlab.com/ultimaker/embedded/platform/ulticontroller_interface_tester}"
-CI_REGISTRY_IMAGE_TAG="${CI_REGISTRY_IMAGE_TAG:-latest}"
-CROSS_COMPILE="${CROSS_COMPILE:-arm-linux-gnueabihf-}"
+ARCH="${ARCH:-arm64}"
```

Comment:
lately, I've switched to building with qemu-user-static, and for 'small' programs, this is not too bad in terms of overhead.

See here an example, which is a bit much :p and I'm not fully happy with it yet, but time ...: https://gitlab.com/esbs/bootstrap/-/blob/master/.gitlab-ci.yml

The magic is mostly made possible with the https://gitlab.com/esbs/bootstrap/-/blob/master/Dockerfile#L11 `TARGET_ARCH` flag in your container.